File: DVLD_Project/ReTakeExam.vb
Imports System
Imports System.Windows.Forms

Namespace DVLD_Project
    Public Partial Class ReTakeExam
        Inherits UserControl
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub LoadControl(NumberOfExamsBefor As Integer, ExamFees As Single)
            If NumberOfExamsBefor = 0 Then
                lblFeesR.Text = "0"
                lblFeesT.Text = ExamFees.ToString()
            Else
                lblFeesR.Text = "5"
                lblFeesT.Text = (5F + ExamFees).ToString()
            End If
        End Sub

        Public Function GetTotalFees() As Single
            Return Single.Parse(lblFeesT.Text.ToString())
        End Function

        Private Sub ReTakeExam_Load(sender As Object, e As EventArgs)

        End Sub
    End Class
End Namespace

File: DVLD_Project/ReTakeExam.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DVLD_Project
{
    public partial class ReTakeExam : UserControl
    {
        public ReTakeExam()
        {
            InitializeComponent();
        }

        public void LoadControl(int NumberOfExamsBefor,float ExamFees)
        {
            if (NumberOfExamsBefor==0)
            {
                lblFeesR.Text = "0";
                lblFeesT.Text = ExamFees.ToString();
            }
            else
            {
                lblFeesR.Text = "5";
                lblFeesT.Text = (5f+ExamFees).ToString();
            }
        }

        public float GetTotalFees()
        {
            return float.Parse(lblFeesT.Text.ToString());
        }

        private void ReTakeExam_Load(object sender, EventArgs e)
        {

        }
    }
}
